index.js

```diff
@@ -31,16 +32,18 @@ module.exports = function (archive, target, opts, cb) {
   var entries = {}
   var watcher
 
-  if (opts.live) {
+  if (opts.watch && archive.live) {
     watcher = chokidar.watch([target], {
       persistent: true,
       ignored: opts.ignore
     })
     watcher.once('ready', function () {
       watcher.on('add', function (file, stat) {
+        status.emit('file watched', {path: file, mode: 'created'})
```

Comment:
OK, yeah that sounds useful! I'm only a bit concerned about the event name, because it should be saying that has begun being watched, not that it was watched..unless I confuse my english here? What about `file start watching`